### ISSUE TITLE
Loss Distribution at 400k and 600k steps - OWT - 124M model

### ISSUE THREAD

**Issue by MarkoKarbevski** [karpathy/nanoGPT#644]
What is a reasonable interval of the losses at 400k and 600k steps for the default ~124M model of the repo on OWT?

My val loss is around 2.95 at 400k and 2.90 at 600k steps.
Should I be concerned that I might be doing something wrong, or is this reasonable?

I am asking because the plotted val loss is around 2.905 in Karpathy's run at 400k steps.

**Comment by MarkoKarbevski** [comment]
> > > > Should I be concerned that I might be doing something wrong, or is this reasonable?
> > > > I am asking because the plotted val loss is around 2.905 in Karpathy's run at 400k steps.
> > > 
> > > 
> > > No you shouldn't be concerned because it is generally due to initialization of the weights. If you are using same seed as Karpathy did then it could due to either due to the tokenization, dataset or some hyperparameters.
> > > To put it simply there's nothing wrong, it's totally fine and reasonable. Also everything should be fine until your model is generating coherent enough text. If not then there's some problem.
> > 
> > 
> > I am training it (against another variant) for scientific purposes, but was wondering if I may have made a mistake to get that type of a loss
> 
> Nope, I don't think you made any mistake. It's normal to get that type of loss. Good luck with your project.

Actually I did have a mistake, the loss should be very close to 2.905 (+- 1% at worst) at that step 😅 but thank you for your support in any case
